pkg/operator/mon/health.go

```diff
@@ -118,6 +124,27 @@ func (c *Cluster) checkHealth() error {
 		}
 	}
 
+	if len(monsNotFound) > 0 {
```

Comment:
could you add a unit test that hits this case? an example of a failover test is in `TestCheckHealth`

Reply:
On it.